src/libs/logging/src/lib.rs

```diff
@@ -3,26 +3,50 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#[macro_use]
+extern crate lazy_static;
+use arc_swap::ArcSwap;
 use slog::{o, record_static, BorrowedKV, Drain, Key, OwnedKV, OwnedKVList, Record, KV};
+
 use std::collections::HashMap;
 use std::io;
 use std::io::Write;
 use std::process;
 use std::result;
-use std::sync::Mutex;
+use std::sync::{Arc, Mutex};
 
 mod file_rotate;
 mod log_writer;
 
 pub use file_rotate::FileRotator;
 pub use log_writer::LogWriter;
 
+lazy_static! {
+    pub static ref LOGGERS: ArcSwap<HashMap<String, slog::Logger>> =
+        ArcSwap::from(Arc::new(HashMap::<String, slog::Logger>::new()));
```

Comment:
where did you init the vmm-dragonball logger for LOGGERS, I didn't find it.

Reply:
In file `src/runtime-rs/crates/shim/src/loggers.rs`. Forgot to stage this change, my bad.